src/main/java/com/github/fabienrenaud/jjb/stream/Deserialization.java

```diff
@@ -383,4 +383,177 @@ static User toUser(Value v) {
             return user;
         }
     }
+
+    @Benchmark
+    @Override
+    public Object antons() throws Exception {
+        return antons(sk.antons.json.parse.JsonScanner.instance(JSON_SOURCE().nextString()));
+    }
+
+
+
+    public Users antons(sk.antons.json.parse.JsonScanner scanner) throws IOException {
```

Comment:
Please move user deserialization logic to the `UsersStreamDeserializer`

Note: looks like I'll have to move PureJson's code out there too.

Reply:
done